plugins/marketplace-ops/scripts/score-items.py

```
@@ -0,0 +1,285 @@
+    return protected
+
+
+def is_protected(path, protected):
+    for prefix in protected:
+        if path.startswith(prefix) or path.rstrip("/") == prefix.rstrip("/"):
+            return True
+    return False
+
+
+def git(repo_root, *args):
+    result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)